### PR TITLE
doctl-compute-droplet: add page

### PR DESCRIPTION
**Pull request by 258204**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #7475

### CODE REVIEW

File: pages/common/doctl-compute-droplet.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# doctl compute droplet
+
+> List create and delete virtual machines called droplets.
```
Comment (CleanMachine1) [inline pages/common/doctl-compute-droplet.md:3]:
```suggestion
> List, create, and delete virtual machines which are called droplets.
```

File: pages/common/doctl-compute-droplet.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/compute/droplet/>.
+
+- Create a droplet:
+
+`doctl compute droplet create --region {{region}} --image {{OS_image}} --size {{vps_type}} {{droplet_name}}`
+
+- Delete a droplet:
+
+`doctl compute droplet delete {{droplet_id}}`
```
Comment (CleanMachine1) [inline pages/common/doctl-compute-droplet.md:12]: is the droplet_id == droplet_name?

File: pages/common/doctl-compute-droplet.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- List Droplets:
```
Comment (CleanMachine1) [inline pages/common/doctl-compute-droplet.md:14]:
```suggestion
- List droplets:
```

why capital on just this one?

File: pages/common/doctl-compute-droplet.md (hunk 2 above)
Comment (258204) [reply pages/common/doctl-compute-droplet.md:12]:
```suggestion
`doctl compute droplet delete {{droplet_id|droplet_name}}`
```

**Review by sbrl — APPROVED** [review]
Thanks, @258204!

File: pages/common/doctl-compute-droplet.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# doctl compute droplet
+
+> List, create, and delete virtual machines which are called droplets.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/compute/droplet/>.
+
+- Create a droplet:
+
+`doctl compute droplet create --region {{region}} --image {{OS_image}} --size {{vps_type}} {{droplet_name}}`
```
Comment (navarroaxel) [inline pages/common/doctl-compute-droplet.md:8]: We use `snake_case` for placeholders.
```suggestion
`doctl compute droplet create --region {{region}} --image {{os_image}} --size {{vps_type}} {{droplet_name}}`
```

**Review by navarroaxel — APPROVED** [review]
I committed the small fix, and I proceed to merge this PR.

**Pull request merged**